models/eval/inspirational_generation.py

```diff
@@ -200,11 +201,16 @@ def gradientDescentOnInput(model,
     gradientDecay = 0.1
 
     nImages = input.size(0)
+    assert nImages == 1
+    # assert randomSearch
+    thelosses = [1., 3., 3.]  # These numbers should be discussed...
+    target = MultiobjectiveFunction(lambda x: thelosses, tuple(thelosses))
```

Comment:
this is constant? maybe use lambda x:x? and then target(losses) instead of target.compute_aggregate_loss?

Reply:
When thelosses will be updated, this function will return something different. Tested with:
y=[1]

def biz(x):
    print(y)
    return y

biz(2)
biz(2)
y=[3]
biz(2)

Reply:
I would have though that overriding y would unlink the external y from the one used internally but indeed...
Still, this is an ugly hack you should reaaaaaaally avoid doing for both clarity and avoid extremely weird bugs

Reply:
The point is that Nevergrad is not equipped for Ask&Tell in multiobjective mode. I did not find a better solution for now.